src/routes/about/+page.svelte

```diff
@@ -1 +1,260 @@
+			<div class="relative grid grid-cols-1 top-[16rem] laptop:top-0 laptop:left-[3rem] laptop:top-[5rem] z-10 bg-body-background-blue pl-7 pr-4 py-7 laptop:p-12 rounded-l-lg laptop:rounded-lg mb-12 w-11/12 min-w-[310px] h-[380px] laptop:w-[460px] laptop:h-auto">
+				<h1 class="text-white text-lg tablet:text-2xl laptop:text-3xl font-gotham-medium laptop:pb-4 ">
+					ColorStack UF is a team committed to...
+				</h1>
+				<p class="text-white font-archer text-lg tablet:text-xl leading-6 self-start">
+					Investing in the futures of those left behind every day. We’re a tech student organization
+					cultivating the strongest campus community of its kind to deliver community building,
+					academic support, and career development opportunities.
+                </p>
+				<a href="https://linktr.ee/colorstackuf" target="_blank" class="self-end laptop:self-start">
+					<div class="text-black w-44 text-center bg-colorstackuf-orange transition-bg-color transition-color hover:bg-colorstackuf-blue hover:text-white duration-300 py-3 px-7 rounded-3xl mt-8 font-gotham-medium">
+						View Link Tree
+					</div>
+				</a>
+			</div>
+		</div>
+	</div>
+
+	<!-- About ColorStack National -->
+	<div class="mt-60 laptop:mt-0">
+		<div class="laptop:bg-body-background-white bg-body-background-blue lg-desktop:max-w-[1920px] mx-auto laptop:py-40 py-6 px-8">
+			<div class="bg-body-background-blue rounded-lg laptop:p-16">
+				<div class="relative grid grid-cols-1 laptop:col-start-2 laptop:grid-cols-2 gap-x-12">
+					<img src="graphics/colorstack_students.png" alt="People" class="w-full rounded-lg" />
+
+					<div class="flex flex-col items-center text-center">
+						<h1 class="text-white text-xl laptop:text-4xl pt-8 font-gotham-medium pb-4 laptop:pb-12">
+							About ColorStack National
+						</h1>
+						<p class="text-white font-archer text-lg">
+							The ColorStack Family is our community of 10,000+ Black and Latinx Computer Science
+							students from 900+ schools nationwide. We’re committed to providing community building,
+							academic support, and career development year-round through our active Slack channel,
+							monthly Fam Fridays, and partnerships with 70+ top tech companies.
+						</p>
+						<a href="https://app.colorstack.io/apply" target="_blank">
+							<button class="text-black text-center bg-colorstackuf-orange transition-bg-color transition-color hover:bg-colorstackuf-blue hover:text-white duration-300 py-4 px-7 rounded-3xl mt-8 laptop:mt-16 font-gotham-medium">
+								Become a National Member
+							</button>
+						</a>
+					</div>
+				</div>
+			</div>
+		</div>
+	</div>
+
+	<!-- Eboard Grid -->
+	<div class="bg-body-background-blue">
+		<div class="lg-desktop:max-w-[1920px] mx-auto py-12 laptop:px-12 px-6">
+			<div class="flex flex-col justify-center items-center font-gotham-medium pb-12">
+				<h1 class="text-white text-[2rem] text-center">
+					Meet the E-Board
```

Comment:
Align each picture to the start of its container. Otherwise, the pictures become uneven when names overflow.

<img width="329" alt="image" src="https://github.com/user-attachments/assets/6aaf1fe8-0621-4aa0-aab2-ed9378318439">